### DISCUSSION TITLE
In docker compose, which volume is immich storing photos data?

### DISCUSSION THREAD

**Discussion by douglasg14b** [immich-app/immich#8337] [Q&A] [answered]
The docs make it sound like `UPLOAD_LOCATION` is a backup location:

> Populate UPLOAD_LOCATION with your preferred location for storing backup assets.

Where is the data the app uses for the app stored? Or is the same location as the backups location?

**Comment by bo0tzz** [comment, marked as answer]
All the assets are stored in `UPLOAD_LOCATION`. Metadata, user settings, etc are in the postgres database which is stored in the `pgdata` volume.
